backends/arm/test/models/test_mobilenet_v2_arm.py

```diff
@@ -30,37 +30,36 @@ class TestMobileNetV2(unittest.TestCase):
         "executorch_exir_dialects_edge__ops_aten_convolution_default",
     }
 
-    @unittest.skip("This test is not supported yet")
+    # TODO: MI test case have numerical differences
     def test_mv2_tosa_MI(self):
         (
             ArmTester(
                 self.mv2,
                 inputs=self.model_inputs,
                 profile=TosaProfile.MI,
                 backend=ArmBackendSelector.TOSA,
+                permute_memory_to_nhwc=True,
             )
             .export()
-            .check(list(self.all_operators))
             .to_edge()
+            .check(list(self.all_operators))
             .partition()
             .to_executorch()
-            .run_method()
-            .compare_outputs()
         )
 
-    @unittest.skip("This test is not supported yet")
     def test_mv2_tosa_BI(self):
         (
             ArmTester(
                 self.mv2,
                 inputs=self.model_inputs,
                 profile=TosaProfile.BI,
                 backend=ArmBackendSelector.TOSA,
+                permute_memory_to_nhwc=True,
             )
             .quantize()
             .export()
-            .check(list(self.all_operators))
             .to_edge()
+            .check(list(self.all_operators))
```

Comment:
this needs updated to remove the check of `executorch_exir_dialects_edge__ops_aten__native_batch_norm_legit_no_training_default`